recipe/bld.bat

```diff
@@ -1,93 +1,76 @@
 )
-if "%VSTRING%" == "14" (
-    set GENERATOR=Visual Studio 14 2015
-    set OPENCV_VC=vc14
-)
-if not defined GENERATOR EXIT 1
 
-if %ARCH% EQU 64 (
-    set GENERATOR=%GENERATOR% Win64
-    set OPENCV_ARCH=x64
+for /F "tokens=1,2 delims=. " %%a in ("%PY_VER%") do (
+   set "PY_MAJOR=%%a"
+   set "PY_MINOR=%%b"
 )
+set PY_LIB=python%PY_MAJOR%%PY_MINOR%.lib
 
-rem I had to take out the PNG_LIBRARY because it included
-rem a Windows path which caused it to be wrongly escaped
-rem and thus an error. Somehow though, CMAKE still finds
-rem the correct png library...
-cmake .. -LAH -G"%GENERATOR%"                       ^
-    -DBUILD_TESTS=0                                 ^
-    -DBUILD_DOCS=0                                  ^
-    -DBUILD_PERF_TESTS=0                            ^
-    -DBUILD_ZLIB=0                                  ^
-    -DBUILD_TIFF=0                                  ^
-    -DBUILD_PNG=0                                   ^
-    -DBUILD_OPENEXR=1                               ^
-    -DBUILD_JASPER=1                                ^
-    -DBUILD_JPEG=0                                  ^
-    -DPYTHON_EXECUTABLE="%PYTHON%"                  ^
-    -DPYTHON_INCLUDE_PATH="%PREFIX%\include"        ^
-    -DPYTHON_LIBRARY="%PREFIX%\libs\python27.lib"   ^
-    -DPYTHON_PACKAGES_PATH="%SP_DIR%"               ^
-    -DWITH_EIGEN=1                                  ^
-    -DWITH_CUDA=0                                   ^
-    -DWITH_OPENNI=0                                 ^
-    -DWITH_FFMPEG=0                                 ^
-    -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%"
-
-cmake --build . --config %CMAKE_CONFIG% --target ALL_BUILD
-cmake --build . --config %CMAKE_CONFIG% --target INSTALL
-
+cmake .. -LAH -G "NMake Makefiles"                                    ^
+    -DWITH_EIGEN=1                                                    ^
+    -DBUILD_TESTS=0                                                   ^
+    -DBUILD_DOCS=0                                                    ^
+    -DBUILD_PERF_TESTS=0                                              ^
+    -DBUILD_ZLIB=0                                                    ^
+    -DBUILD_opencv_bioinspired=0                                      ^
+    -DBUILD_TIFF=0                                                    ^
+    -DBUILD_PNG=0                                                     ^
+    -DBUILD_OPENEXR=1                                                 ^
+    -DBUILD_JASPER=1                                                  ^
+    -DBUILD_JPEG=0                                                    ^
+    -DWITH_CUDA=0                                                     ^
+    -DWITH_OPENCL=0                                                   ^
+    -DWITH_OPENNI=0                                                   ^
+    -DWITH_FFMPEG=0                                                   ^
+    -DWITH_VTK=0                                                      ^
```

Comment:
We should take a look at this PR ( https://github.com/conda-forge/staged-recipes/pull/453 ) again to see if we can get VTK.